logging/unit_tests/test_client.py

```diff
@@ -562,7 +563,8 @@ def test_get_default_handler_app_engine(self):
                                 credentials=_make_credentials(),
                                 use_gax=False)
 
-        with _Monkey(_MUT, _LOG_PATH_TEMPLATE='{pid}'):
+        temp_log_path = os.path.join(tempfile.mkdtemp(), '{pid}')
```

Comment:
Ya I mentioned that in PR comment, thought it wasn't a big deal cause it's in tmp but I supposed might as well clean it up, ty for fixing.